coalib/bearlib/languages/definitions/shell.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class shell:
+    aliases = 'bash', 'sh'
+
+    extensions = '.sh', '.bash', '.zsh'
```

Comment:
Is there a way to check for `.*sh`, since this doesn't cover KornShell, CShell, etc.